wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -221,32 +128,44 @@ public static void main(String... args) throws Exception {
     return null;
   }
 
+  @Deprecated
   public WireCompiler(String protoPath, List<String> sourceFileNames, List<String> roots,
```

Comment:
Does anybody use this? Could we change the callers to use String args and delete this?